documentation/vesting.md

```diff
@@ -10,20 +10,21 @@ The following table gives an overview of the vesting schedules, each implemented
-| DAO Fund | L2            | 15,000,000 | <dao treasury> | 1716163200 | 226 | 6,250,000 LSK liquid at migration, 8,750,000 LSK are linearly released in 2024, starting 20.05.2024. |
-| DAO Fund | L2            | 30,000,000 | <dao treasury> | 1735689600 | 730 | 15,000,000 LSK are linearly released in 2025, 15,000,000 LSK are linearly released in 2026. |
-| DAO Fund - optional | L2            | 90,000,000 | <dao treasury> | 1798761600 | 2192 | 15,000,000 LSK are linearly released in the years 2027-2032. |
-| DAO Fund - optional | L2            | 10,000,000 | <dao treasury> | 1988150400 | 365 | 10,000,000 LSK are linearly released in 2033. |
+| Category | Network Layer | Amount (LSK) | Wallet address | Beneficiary Address | Start Timestamp | Duration in Days | Description |
+|----------|---------------|--------------|----------------|---------------------|----------|------------------|-------------|
+| Treasury | L1            | 8,000,000 |[0x18a0b8c653c291D69F21A6Ef9a1000335F71618e](https://eth.blockscout.com/address/0x18a0b8c653c291D69F21A6Ef9a1000335F71618e)| [0xCAaCF7d9E40D0f4dB66419d678A8D46dE74B0C02](https://eth.blockscout.com/address/0xCAaCF7d9E40D0f4dB66419d678A8D46dE74B0C02) | 1735689600 | 1095 | All tokens are linearly released over 3 years between 1.1.2025 and 31.12.2027. |
+| Team I    | L1            | 5,500,000 |[0xe09899a4C98473460BC19D136B330608B465Dc55](https://eth.blockscout.com/address/0xe09899a4C98473460BC19D136B330608B465Dc55)| [0x84798151d27C09E9B6C85A110E6a195D83A4D5F0](https://eth.blockscout.com/address/0x84798151d27C09E9B6C85A110E6a195D83A4D5F0) | 1735689600 | 1461 | All tokens are linearly released over 4 years between 1.1.2025 and 31.12.2028. |
+| Team II    | L1            | 1,500,000 |[0x2294A7f24187B84995A2A28112f82f07BE1BceAD](https://eth.blockscout.com/address/0x2294A7f24187B84995A2A28112f82f07BE1BceAD)| [0x586C7735d78f421495FE7e4E32B4e13a90661395](https://eth.blockscout.com/address/0x586C7735d78f421495FE7e4E32B4e13a90661395) | 1735689600 | 1461 | All tokens are linearly released over 4 years between 1.1.2025 and 31.12.2028. |
+| Investors | L1            | 30,000,000 |[0x58a61b1807a7bDA541855DaAEAEe89b1DDA48568](https://eth.blockscout.com/address/0x58a61b1807a7bDA541855DaAEAEe89b1DDA48568)| [0x4B00A4659454013388b39DF9b23F5DbE65Bbc06E](https://eth.blockscout.com/address/0x4B00A4659454013388b39DF9b23F5DbE65Bbc06E) | 1716163200 | 730 | 10,000,000 LSK liquid at migration, the remaining 20,000,000 LSK vested linearly over 24 months, starting 20.05.2024. |
+| Ecosystem Fund | L1            | ~7,860,000 |[0x114cB34b1A0fBBB686E31Bf5542d64A98c42eE23](https://eth.blockscout.com/address/0x114cB34b1A0fBBB686E31Bf5542d64A98c42eE23)| [0xC30B50cdCccEb70b8D87fDda4F08258e8A02539E](https://eth.blockscout.com/address/0xC30B50cdCccEb70b8D87fDda4F08258e8A02539E) | 1735689600 | 0 | 5,000,000 LSK are liquid at migration, the remaining ~2,860,000 LSK are released on 1.1.2025. |
```

Comment:
We know the exact numbers now, `2,711,644` are vested and the total is `7,711,644`.

Reply:
Fixed